cypress/e2e/requestPasswordReset.cy.ts

```diff
@@ -0,0 +1,57 @@
+import { REQUEST_PASSWORD_RESET_PATH } from '../../src/config/paths';
+import {
+  REQUEST_PASSWORD_RESET_EMAIL_FIELD_HELPER_ID,
+  REQUEST_PASSWORD_RESET_EMAIL_FIELD_ID,
+  REQUEST_PASSWORD_RESET_ERROR_MESSAGE_ID,
+  REQUEST_PASSWORD_RESET_SUBMIT_BUTTON_ID,
+  REQUEST_PASSWORD_RESET_SUCCESS_MESSAGE_ID,
+} from '../../src/config/selectors';
+import { MEMBERS } from '../fixtures/members';
+
+describe('Request password reset', () => {
+  it('For existing member', () => {
+    cy.setUpApi();
+    cy.visit(REQUEST_PASSWORD_RESET_PATH);
+    // request password reset for an existing member
+    cy.get(`#${REQUEST_PASSWORD_RESET_EMAIL_FIELD_ID}`).type(
+      MEMBERS.GRAASP.email,
+    );
+    cy.get(`#${REQUEST_PASSWORD_RESET_SUBMIT_BUTTON_ID}`).click();
+    cy.get(`#${REQUEST_PASSWORD_RESET_SUCCESS_MESSAGE_ID}`).should(
+      'be.visible',
+    );
+  });
+  it('For non-email', () => {
+    cy.setUpApi();
+    cy.visit(REQUEST_PASSWORD_RESET_PATH);
+
+    cy.get(`#${REQUEST_PASSWORD_RESET_EMAIL_FIELD_ID}`).type(
+      MEMBERS.WRONG_EMAIL.email,
+    );
+
+    // click the button to trigger the validation
+    cy.get(`#${REQUEST_PASSWORD_RESET_SUBMIT_BUTTON_ID}`).click();
+    cy.get(`#${REQUEST_PASSWORD_RESET_SUBMIT_BUTTON_ID}`).should('be.disabled');
+
+    cy.get(`#${REQUEST_PASSWORD_RESET_EMAIL_FIELD_HELPER_ID}`).should(
+      'contain.text',
+      'This does not look like a valid email address',
```

Comment:
I saw this at multiple place, but I put my comment once.

It's more a question than a suggestion, but I'm wondering if we should not use constants instead of text here. If we decide to change the text, this will break the test. But maybe it's exactly what we want? I'm not sure, so I just ask ^^

Reply:
Yes, I think it is better to have the test break if we change the text, because it makes the test more dependant on what the user sees and less on how we coded it. this means we can change the name of the translation key and it does not break the test, because what the user sees has not changed.

Reply:
I know this is something new, we are not really used to doing that, but maybe we can discuss about transitioning to this approach for the future ? Testing what the user actually sees without depending too much on how it is built.